.github/workflows/OCV-PR-5.x-ARM64.yaml

```diff
@@ -13,7 +13,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DOPENCV_DOWNLOAD_PATH=/home/ci/binaries_cache -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON'
+  EXTRA_CMAKE_OPTIONS: '-DBUILD_DOCS=ON -DPYTHON_DEFAULT_EXECUTABLE=/usr/bin/python3 -DOPENCV_DOWNLOAD_PATH=/home/ci/binaries_cache -DBUILD_EXAMPLES=ON -DOPENCV_ENABLE_NONFREE=ON -DWITH_OPENCL=OFF'
```

Comment:
`WITH_OPENCL` is `ON` by default.

We should check that this "default" mode doesn't crash even without OpenCL devices or any OpenCL runtime.

`WITH_OPENCL=OFF` makes sense for dedicated debug ""no-optimization"  builds only.

Reply:
>  this "default" mode doesn't crash even without OpenCL devices or any OpenCL runtime.

Sometimes it causes accuracy problem if we do some changes on the CPU code (like in my warp functions optimization PRs). The changes are all about CPU side, but the OCL tests failed and needs to be changed even though OpenCL code is not touched at all.